package.json

```diff
@@ -504,6 +504,14 @@
           "type": "string",
           "description": "Color of the search highlight text."
         },
+        "vim.searchMatchColor": {
```

Comment:
I think these should be described as "background" and "foreground" in setting names and descriptions

Reply:
I chose the names for consistency with `vim.searchHighlightColor` / `vim.searchHighlightTextColor` and `vim.highlightedyank.color` / `vim.highlightedyank.textColor`. I agree `vim.searchMatchBackground` / `vim.searchMatchForeground` would be clearer, but is it worth breaking convention?

Reply:
Then again, I just noticed all the easymotion colors use the background/foreground convention, so maybe it's not so radical

Reply:
Yeah unfortunately our config schema isn't super consistent

Reply:
Sorry, actually I think this should mirror `vim.searchHightlightColor` and `vim.searchHighlightTextColor`. That consistency is more important than with easymotion or my personal naming preference